lib/cell/i18n.rb

```diff
@@ -0,0 +1,16 @@
+module Cell::I18n
+
+  def t(*args)
+    options   = args.last.is_a?(Hash) ? args.pop.dup : {}
+    key       = args.shift
+    cell_name = self.class.to_s.delete('Cell').underscore
```

Comment:
This is deleting too much from the name (e.g. `'MenuCell'.delete('Cell') # => 'Mnu')` ... I'd suggest e.g. `gsub(/Cell\Z/, '')` instead.